### PR TITLE
Greenbranch

### PR DESCRIPTION
**Pull request by jzoon**
"As an agent
I want to build green constructions
to improve my indicator score"

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
Is there a test for this new module?

File: TU-Agent/tygron.mod2g [hunk 1]
```diff
@@ -20,11 +21,15 @@ module tygron {
...
 	%TODO: We could also buy land to create buildable land
 	if goal(createLandToBuild)
-		then demolish.	
+		then demolish.
 	
 	%If we have the goal to build high buildings, we should enter the build module
 	if goal(indicatorGoal('Bouwen hoge gebouwen', _))
 		then buildConstruction.
+		
+	%If we have the goel to build green things, we should enter the buildGreen module
```
Comment (tvleest) [inline TU-Agent/tygron.mod2g]: goal

Comment (tvleest) [inline TU-Agent/tygron.mod2g]: things -> constructions

**Comment by tvleest** [comment]
Looks good, but we might need to change this when the new custom build action is merged.

**Comment by jzoon** [comment]
Fixed the comment mistakes. There are no tests yet

**Comment by jzoon** [comment]
Tests look good. I think this can be merged if one more person checks the tests

**Comment by daanwerf** [comment]
Approved.

**Pull request merged**